integration/up/hybridmode_test.go

```diff
@@ -37,9 +38,21 @@ dev:
     context: svc
     namespace: user
     mode: hybrid
-    command: "{{ .Shell }} checker.sh"
+    command: ./checker.sh
```

Comment:
I think we need the shell for windows tests. Probably @AdrianPedriza can provide more information about this but we saw that if we used `./script.sh` or `sh script.sh` wasn't working

Reply:
For now I've set `bash` to simplify this test, until we can remove it entirely in the future.